l5kit/l5kit/environment/ppo.py

```diff
@@ -0,0 +1,99 @@
+import argparse
```

Comment:
This file is an example on how to use PPO, and it is also a script. Users install `l5kit` with `pip install l5kit`, so how do they get access to this script ? A better way to structure it would be to move this to the examples folder.